db/migrate/20220131083110_create_gdynia_extra_data.rb

```diff
@@ -0,0 +1,8 @@
+class CreateGdyniaExtraData < ActiveRecord::Migration[7.0]
+  def change
+    create_table :gdynia_extra_data do |t|
```

Comment:
I'm not sure if having city-specific tables is a good idea, but I guess this is just a stub for future development?

Reply:
Pretty much so. For now this data comes only from one city and looking how hard to calculate are most of these factors we are unlikely to get any more data in this format. In case we do it can be reinterpreted as a data format that originates from Gdynia